src/kmeans.jl

```diff
@@ -71,7 +87,7 @@ function _kmeans!{T<:AbstractFloat}(
     maxiter::Int,                   # in: maximum number of iterations 
     tol::Real,                      # in: tolerance of change at convergence 
     displevel::Int)                 # in: the level of display
-
+    
```

Comment:
Trailing whitespace. I don't know what editor you use, but I think Atom trims trailing whitespace by default, and in Vim you can do `:%s/\s\+$//g` to remove it.